assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -53,6 +53,10 @@ export default function UserInputKeywords( { slug, max } ) {
 		values.push( '' );
 	}
 
+	// Store values in local state to prevent
+	// https://github.com/google/site-kit-wp/issues/2900#issuecomment-814843972.
+	const [ localValues, setLocalValues ] = useState( [ '' ] );
```

Comment:
Shouldn't the initial state be `values`?

Reply:
Oops, yeah, that'd be fine 👍🏻

Reply:
And as I suspected, that fixes the bug too. Thanks! 👍🏻